### DISCUSSION TITLE
[Enhancement]: Can we add a feature to use GPT Vision API?

### DISCUSSION THREAD

**Discussion by garciaraul85** [Pythagora-io/gpt-pilot#441] [Ideas]
Can we add a feature to use GPT Vision API and create a UI/UX agent that takes blue-prints from the user to create UI/UX?
I think that would be dope. Like you upload a sketch from a paper drawing to a professional blue-print of the UI whether its web, mobile or desktop and we let the agents code the UI/UX and finally we can add DALLE-2 or 3 or Midjourney create the logos and images for the UI/UX.

**Comment by tristdrum** [comment]
Fantastic idea!
